frontend/src/app/modules/fields/edit/field-types/te-work-package-edit-field.component.ts

```diff
@@ -35,14 +35,16 @@ import {
   TimeEntryWorkPackageAutocompleterMode
 } from "core-app/modules/common/autocomplete/te-work-package-autocompleter.component";
 import {HalResource} from "core-app/modules/hal/resources/hal-resource";
+import {InjectField} from "core-app/helpers/angular/inject-field.decorator";
 
 const RECENT_TIME_ENTRIES_MAGIC_NUMBER = 30;
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/0d5d295fec5e57b4542a818c3f681c2be34b856b/frontend/src/app/modules/fields/edit/field-types/te-work-package-edit-field.component.ts#L41"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>